### PR TITLE
upload static files to s3

### PR DESCRIPTION
**Pull request by GopherJ**
close: #39

### CODE REVIEW

File: .github/workflows/s3.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Upload Static Files
+
+on:
+  push:
```
Comment (crystalynliu) [inline .github/workflows/s3.yml:4]: Can we change to trigger it when release? so we have time to check on dev environment

**Comment by GopherJ** [comment]
waiting @woshihahaha to fix ACLs

**Review by crystalynliu — APPROVED** [review]

**Comment by GopherJ** [comment]
@crystalynliu  we still have cors issue:

image

**Comment by GopherJ** [comment]
@crystalynliu ready to be merged, now cors should have been fixed by @woshihahaha

**Review by crystalynliu — APPROVED** [review]

**Pull request merged**